mimic/rest/nova_api.py

```diff
@@ -223,13 +224,21 @@ def _region_collection_for_tenant(self, tenant_id):
     def _image_collection_for_tenant(self, tenant_id):
         tenant_session = self._session_store.session_for_tenant_id(tenant_id)
         image_global_collection = tenant_session.data_for_api(
-            "image_collection",
-            lambda: GlobalImageCollection(tenant_id=tenant_id,
-                                          clock=self._session_store.clock))
+            "nova_image_collection",
+            lambda: GlobalNovaImageCollection(tenant_id=tenant_id,
+                                              clock=self._session_store.clock))
         image_region_collection = image_global_collection.collection_for_region(
             self._name)
         return image_region_collection
 
+    def _image_store_for_tenant(self, tenant_id):
```

Comment:
I know this is a private method, but a docstring will help future maintainers understand what the purpose is.  It would be helpful to add one.